packages/frontend/src/components/campaigns/campaign/chain/index.tsx

```diff
@@ -11,7 +10,7 @@ interface ChainProps {
 }
 
 export function Chain({ id }: ChainProps) {
```

Comment:
Here we should keep the chain id coming from the props, since it's the campaigns's chain id.